### DISCUSSION TITLE
Tab "Embedded Meta Info" missing

### DISCUSSION THREAD

**Discussion by MGS76** [pimcore/pimcore#16485] [Q&A / Support] [answered]
Hello,

I use the latest version 11.1.4. What can be the reason why this tab is not shown? Images have metadata - and it's used for search...

Regards

Michael

**Comment by MGS76** [comment]
I found this in the log file:
Is that the reason?
`[2024-01-18T06:43:44.722015+00:00]` php.INFO: User Deprecated: Since symfony/framework-bundle 6.4: The "annotations.cached_reader" service is deprecated without replacement. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Since symfony/framework-bundle 6.4: The \"annotations.cached_reader\" service is deprecated without replacement. `

**Comment by kingjia90** [comment, marked as answer]
Check If `pimcore_asset_update`is properly set and running


See also:
https://pimcore.com/docs/platform/Pimcore/Getting_Started/Installation/Webserver_Installation#5-maintenance-cron-job:~:text=pimcore_asset_update
(https://github.com/pimcore/pimcore/blob/757375677dc83a44c6c22f26d97452cc5cda5d7c/lib/Messenger/Handler/AssetUpdateTasksHandler.php#L138)

**Reply by MGS76** [reply]
Thank you - that was the problem - job didn't run!

**Comment by MGS76** [comment]
Problem is solved with kingjia90 answer - pimcore_asset_update was not running
